src/universal/styles/theme/globalStyles.js

```diff
@@ -81,6 +81,16 @@ export default `
     font-weight: 600;
   }
 
+  @media print {
```

Comment:
oh look at you. You printing your task board for posterity?

Reply:
Ah, no I see you're just moving where the style is declared to here... 👍

Reply:
haha yeah, i think we included that for invoices way back when